user/caribou/module/include/CaribouEvent2StdEventConverter.hh

```diff
@@ -36,12 +40,26 @@ namespace eudaq {
     void decodeChannel(const size_t adc, const std::vector<uint8_t>& data, size_t size, size_t offset, std::vector<std::vector<uint16_t>>& waveforms, uint64_t& timestamp) const;
     static size_t trig_;
```

Comment:
The `trig_` could be removed with a) reading it from the FPGA event ID and b) the new filter in Corry upstream?

Reply:
a) I do not think we have that for this device. One could use the trigger ID from the EUDAQ event, but this would not work for our case since we store data from 2 devices in one file.
b) The other way around. This works only with the new filter in corry, since we do not increment it from the converter for the other device.
c) Since time stamps are fine now, all of that does not really matter anymore.

Reply:
d) excellent. Thanks!